### PR TITLE
Mb 14733 customer can see closeout office on ppm shipment review card

### PR DESCRIPTION
**Pull request by dymonmoore**
## Summary
This PR displays a user's closeout office location selection on the PPMShipment Review Card. The closeout office field should be shown in the following conditions:

<br class="Apple-interchange-newline">

Army/Air Force | Marines/Coast Guard/Navy
 -- | --
If the customer selects Army or Air Force as their service, they’ll see the Closeout location in the shipment card | If the customer selects Marines, Coast Guard, or Navy as their service, they WILL NOT see the Closeout location in the shipment card
- The closeout location should be the same for all PPM shipments being submitted on the move
- If the customer closeout location is changed, it must be updated on the PPMShipment Card
### Screenshots
Screen Shot 2023-01-09 at 2 15 48 PM

## Setup to Run Your Code

<details>
<summary>💻 You will need to use three separate terminals to test this locally.</summary>

##### Terminal 1

Start the Storybook locally.

```sh
make storybook
```

##### Terminal 2

Start the UI locally.

```sh
make client_run
```

##### Terminal 3

Start the Go server locally.

```sh
make server_run
```

</details>

## Verification Steps for Author

These are to be checked by the author.
### Step One
- Verify that the logged_in API response includes the closeout_office location:
- Open developer tools network tab
- Use this login for customer app: closeoutoffice@ppm.closeout
- Inspect network results for 'logged_in' api response
- Check service_member > orders > moves in payload for 'closeout_location'
- Visually verify closeout office field is displayed on the PPMShipment Card
### Step Two 
- Verify that the logged_in API response does not include the closeout_office location:
- Open developer tools network tab
- Use this login for customer app: 2needcloseout@ppm.closeout (milmove)
- Inspect network results for 'logged_in' api response
- Check service_member > orders > moves in payload for 'closeout_location' is undefined 
- Visually verify closeout office field is **NOT** displayed on the PPMShipment Card

## Verification Steps for Reviewers

These are to be checked by a reviewer.

### Step One
- Use this login for customer app: closeoutoffice@ppm.closeout
- Click review your request
- Visually verify closeout office field is displayed on the PPMShipment Card
### Step Two
- Use this login for customer app: 2needcloseout@ppm.closeout (milmove)
- Click review your request
- Visually verify closeout office field is **NOT** displayed on the PPMShipment Card

### CODE REVIEW

**Review by dinabeigelman — COMMENTED** [review]
Took a look and I'm not seeing the Closeout office in the shipment card on the review page when the customer enters an Army or Air Force move.

This is working as expected for Coast Guard (no Closeout office listed on review page).

**Comment by dymonmoore** [comment]
> Took a look and I'm not seeing the Closeout office in the shipment card on the review page when the customer enters an Army or Air Force move.
> 
> This is working as expected for Coast Guard (no Closeout office listed on review page).

Could you check these accounts to see if the closeout office field populates? closeoutoffice@ppm.closeout and readyToFinish2@ppm.approved

File: src/components/Customer/Review/ShipmentCard/PPMShipmentCard/PPMShipmentCard.jsx [hunk 1]
```diff
@@ -117,12 +139,16 @@ PPMShipmentCard.propTypes = {
   showEditAndDeleteBtn: bool.isRequired,
   onEditClick: func,
   onDeleteClick: func,
+  move: shape({}),
```
Comment (duncan-truss) [inline src/components/Customer/Review/ShipmentCard/PPMShipmentCard/PPMShipmentCard.jsx:142]: Sorry I know this was my hack suggestion as we were pairing to not trigger the linter.

Can you add closeout_office to the MoveShape here https://github.com/transcom/mymove/blob/0146acb8d388654f2407d413e614a99be0c0606f/src/types/customerShapes.js#L32 ? You should be able to reuse the [TransportaitonOfficeShape](https://github.com/transcom/mymove/blob/b46b8e1f245174532fd9287eb8abd950ccb89aa3/src/types/user.js#L10) too like:

```javascript
export const MoveShape = shape({
  id: string,
  locator: string,
  selected_move_type: string,
  status: string,
  closeout_office: TransportationShape
});
```

and then update this line to be `move: MoveShape`

**Review by dinabeigelman — APPROVED** [review]
Took a look and I'm seeing the closeout office in both those test data user's shipment cards! Tho the state of the move is after it was submitted. I imagine this shipment card works before the move has been submitted as well? Approving and going to keep an eye out for this (move state) in future PRs this sprint.

**Comment by dymonmoore** [comment]
After a long pair session with Patrick, we figured out Closeout_Office is dropped from the API payload between the review and submission of PPMShipments (lines 96-98). Further action will need to be taken to ensure the closeout_office is included in the API response after submission of  PPMShipments.  

Screen Shot 2023-01-13 at 8 28 32 AM

**Comment by duncan-truss** [comment]
> After a long pair session with Patrick, we figured out Closeout_Office is dropped from the API payload between the review and submission of PPMShipments (lines 96-98). Further action will need to be taken to ensure the closeout_office is included in the API response after submission of PPMShipments.
> 
> Screen Shot 2023-01-13 at 8 28 32 AM

That's a great catch, sorry that's a fetcher I missed but does show the value of our Cypress tests.  I know you're OOO today, I could quickly add in the fetcher work seems like an easy eager loading tweak.

**Review by duncan-truss — APPROVED** [review]

**Pull request merged**